pom.xml

```diff
@@ -28,6 +28,7 @@
     <module>aws-junit</module>
     <module>autoconfigure</module>
     <module>sparkstreaming-stream-kinesis</module>
+    <module>collector-kinesis</module>
```

Comment:
will need a README and autoconfig. I think the other READMEs are pretty nice, so copy/paste away!